### ISSUE TITLE
Accessibility colour fail in a kicker

### ISSUE THREAD

**Issue by paperboyo** [guardian/frontend#8367]
Hello,

I am sorry for ambiguously sounding title, but I do not know how are these new series kickers called...

As seen on UK front ‘on this site’ container (BTW, what’s not on this site? :D):

![colours fail](https://cloud.githubusercontent.com/assets/6032869/6237437/b56af148-b6ea-11e4-9d9f-e8de50f88791.PNG)

story URL: http://www.theguardian.com/commentisfree/2015/feb/17/guardian-view-on-the-church-and-election-talking-sense

Regards
Mateusz

**Comment by paperboyo** [comment]
fixed by https://github.com/guardian/frontend/pull/8483. Thanks, closing.

**Comment by sndrs** [comment]
sorry  -didn't see the issue, shoudl have ref'd it

On Tue, Mar 10, 2015 at 12:47 AM Mateusz <notifications@github.com> wrote:

> Closed #8367 <https://github.com/guardian/frontend/issues/8367>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/issues/8367#event-248995173>.
>
> ------------------------------
> Visit theguardian.com. On your mobile and tablet, download the Guardian
> iPhone and Android apps theguardian.com/guardianapp and our tablet
> editions theguardian.com/editions.  Save up to 57% by subscribing to the
> Guardian and Observer - choose the papers you want and get full digital
> access.  Visit subscribe.theguardian.com
>
> This e-mail and all attachments are confidential and may also be
> privileged. If you are not the named recipient, please notify the sender
> and delete the e-mail and all attachments immediately. Do not disclose the
> contents to another person. You may not use the information for any
> purpose, or store, or copy, it in any way.  Guardian News & Media Limited
> is not liable for any computer viruses or other material transmitted with
> or as part of this e-mail. You should employ virus checking software.
>
> Guardian News & Media Limited is a member of Guardian Media Group plc. Registered
> Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered
> in England Number 908396
>
>
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396
